bosh-stemcell/spec/shellout_types/file_spec.rb

```
@@ -275,24 +275,24 @@ def delete_user_and_group(username, groupname)
-          allow(chroot).to receive(:run).and_return([stdout, '', 0])
+        it "should raise" do
+          allow(chroot).to receive(:run).and_return([stdout, "", 0])
 
-          expect { group_file.group }.to raise_error(RuntimeError, 'group entry is an invalid format: has-no-gid-or-members:x')
+          expect { group_file.group }.to raise_error(RuntimeError, "group entry is an invalid format: has-no-gid-or-members:x")
         end
       end
 
-      context('when the group belonging to the file does not exist') do
+      context("when the group belonging to the file does not exist") do
         let(:current_group) { rand(100) + 1 * 65535 }
```

RuboCop (1.86.0)

[warning] Wrap expressions with varying precedence with parentheses to avoid ambiguity.

(Lint/AmbiguousOperatorPrecedence)